guides/common/modules/proc_configuring-the-freeipa-server-to-use-cross-forest-trust.adoc

```diff
@@ -14,9 +14,17 @@ For example:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-[nss]
-user_attributes=+mail, +sn, +givenname
+[domain/lab.example.com]
```

Comment:
I would have expected this to remain `domain/EXAMPLE` or be `domain/EXAMPLE.COM` or `domain/EXAMPLE.ORG` as we have in other examples (but it should remain capitalized and probably not introduce a "LAB" subdomain which we do not explain)

Reply:
Hello, Yes, we can change it. That was the domain I used during the tests.

Waldirio